nunaserver/nunaserver/generator.py

```diff
@@ -6,17 +6,14 @@
 import os
 import tempfile
 import zipfile
-import logging
+import minio
+import typing
 from typing import List
 from pathlib import Path
 from pydsdl import read_namespace
 from pydsdl._error import InvalidDefinitionError
```

Comment:
Overlooked this part in my previous review.

```suggestion
from pydsdl import read_namespace, InvalidDefinitionError
```